crates/rslint_cli/src/panic_hook.rs

```diff
@@ -1,23 +1,30 @@
 //! The custom panic hook used by the linter to issue a more descriptive explanation.
 
 use crate::{lint_err, lint_note};
+use std::io::{stderr, Write};
 use std::panic::PanicInfo;
 
 pub fn panic_hook(info: &PanicInfo) {
     lint_err!("The linter panicked unexpectedly. this is a bug.");
-    eprintln!("We would appreciate a bug report: https://github.com/RDambrosio016/RSLint/issues/new?labels=ILE%2C+bug&template=internal-linter-error.md\n");
+
+    let stderr = stderr();
+    let mut stderr_lock = stderr.lock();
+
+    writeln!(stderr_lock, "We would appreciate a bug report: https://github.com/RDambrosio016/RSLint/issues/new?labels=ILE%2C+bug&template=internal-linter-error.md\n").expect("panic_hook failed to write to stderr");
+
     lint_note!("Please include the following info: ");
```

Comment:
same as above

Reply:
This would go to `stdout`. However, since there is only one call, I will probably just have a pair of curly braces to contain the lock. Or should I expand the lock's scope to the entire function?

Reply:
You can simply swap the note with a writeln! like the others